protoc-gen-orion/orion.go

```diff
@@ -96,13 +97,21 @@ type orionMiddleware struct {
 	Names      string
 }
 
+type ProtocParams struct {
+	StandAloneMode      bool
```

Comment:
change to `StandaloneMode`, because standalone is a single word

Reply:
fixed.